cmake/ActsExternSources.cmake

```diff
@@ -31,7 +31,7 @@ set( ACTS_FRNN_SOURCE
 mark_as_advanced( ACTS_FRNN_SOURCE )
 
 set( ACTS_GEOMODEL_SOURCE
-   "GIT_REPOSITORY;https://gitlab.cern.ch/GeoModelDev/GeoModel;GIT_TAG;4.6.0;" CACHE STRING "Source to take GeoModel from")
+   "GIT_REPOSITORY;https://gitlab.cern.ch/GeoModelDev/GeoModel;GIT_TAG;${_acts_geomodel_version};" CACHE STRING "Source to take GeoModel from")
```

Comment:
Question: I am operating on Fred's #3117 and he seems to have insisted on removing the trailing semicolon in this source string; do you know if this is a potential problem or if the semicolon should remain?

Reply:
happy to try dropping it